docs/src/components/QuickNav/QuickNav.tsx

```diff
@@ -252,8 +252,8 @@ function onMounted(ref: React.RefObject<HTMLDivElement | null>) {
   };
 }
 
-export function Title({ className, ...props }: React.ComponentProps<'h2'>) {
-  return <div aria-hidden className={clsx('QuickNavTitle', className)} {...props} />;
```

Comment:
Not aria-hidden anymore since this element is announced as almost the first thing when entering the page contents  (it matches the page title; MDN uses an h2 there but I disagree with it)